test/formatting.test.ts

```diff
@@ -256,26 +256,35 @@ describe('formatting', () => {
 
   it('Correctly formats an address', () => {
     expect(util.formatAddress(null)).toBe('--, --, -- --');
-    expect(util.formatAddress({address1: '123 Fake St', address2: 'Apt 1-D', city: 'New York', state: 'NY', zip_code: '10001'})).toBe('123 Fake St Apt 1-D, New York, NY 10001');
-    expect(util.formatAddress({address1: '', address2: 'Apt 1-D', city: 'New York', state: 'NY', zip_code: '10001'})).toBe('-- Apt 1-D, New York, NY 10001');
-    expect(util.formatAddress({address1: '123 Fake St', address2: '', city: 'New York', state: 'NY', zip_code: '10001'})).toBe('123 Fake St, New York, NY 10001');
-    expect(util.formatAddress({address1: '123 Fake St', address2: 'Apt 1-D', city: '', state: 'NY', zip_code: '10001'})).toBe('123 Fake St Apt 1-D, --, NY 10001');
-    expect(util.formatAddress({address1: '123 Fake St', address2: 'Apt 1-D', city: 'New York', state: '', zip_code: '10001'})).toBe('123 Fake St Apt 1-D, New York, -- 10001');
-    expect(util.formatAddress({address1: '123 Fake St', address2: 'Apt 1-D', city: 'New York', state: 'NY', zip_code: ''})).toBe('123 Fake St Apt 1-D, New York, NY --');
+    expect(util.formatAddress(
```

Comment:
why was all the formatting changed?

Reply:
Lines were over 120

Reply:
This should be enforced by tslint but isn't, and don't know why it wasn't.